js/index.js

```diff
@@ -152,4 +165,22 @@ function setIdValue(id, value) {
 window.addEventListener("DOMContentLoaded", function() {
     window.location.href = '#meal-modal';
     document.getElementById("loadMealButton").addEventListener('click', mealLoaded);
+    document.getElementById("drop-down-list").onchange = function()
+    {
+        var currentLab = document.getElementsByClassName("lab-selected")[0].innerHTML;
+        var sheet=document.getElementById("drop-down-list").value;
+
+        if(sheet=="first"){
+            populateIngredients(allRecipes[currentLab][0]);
+        }
+        else if(sheet=="second"){
+            populateIngredients(allRecipes[currentLab][1]);
+        }
+        else if(sheet=="third"){
+            populateIngredients(allRecipes[currentLab][2]);
+        }
+        else{
+            populateIngredients(allRecipes[currentLab][3]);
+        }
+    }
```

Comment:
why are the dropdown list values "first" "second" "third" and other, instead of just setting them to be "0" "1" "2" "3" and using that as the index into the array? You can use parseInt("0") to convert the string into a number and then just say `var index = parseInt(sheet) || 0; populateIngredients(allRecipes[currentLab][index]);`